pubspec.yaml

```diff
@@ -8,11 +8,11 @@ dependencies:
     sdk: flutter
   mapbox_gl_platform_interface:
     git:
-      url: https://github.com/tobrun/flutter-mapbox-gl.git
+      url: https://github.com/arsenikstiger/flutter-mapbox-gl.git
       path: mapbox_gl_platform_interface
   mapbox_gl_web:
     git:
-      url: https://github.com/tobrun/flutter-mapbox-gl.git
+      url: https://github.com/arsenikstiger/flutter-mapbox-gl.git
```

Comment:
revert this

Reply:
oh yes, mandatory to make it work on my own project, but now I can revert this for including into your code !